src/problem1.rb

```diff
@@ -0,0 +1,7 @@
+class Problem1
+  def even_numbers_sum(array)
```

Comment:
In case the array is nil, i have to return 0, otherwise the 'select' function does not take nil as a parameter => error. And to return nil from a function is bad.